openmc/cell.py

```diff
@@ -228,6 +233,10 @@ def distribcell_paths(self):
     def volume(self):
         return self._volume
 
+    @property
+    def num_instances(self):
+        return self._num_instances
```

Comment:
Maybe we should raise an exception here if `_num_instances is None` and tell the user they need to run the `count_cell_instances` method.